docs/meshstack.tag-schema.md

```diff
@@ -25,7 +25,124 @@ such metadata include:
 
 Operators can model this metadata as **tags** using a [tag schema](#tag-schemas). When a meshStack implementation has configured [tag schemas](#tag-schemas) it makes metadata tags available to orchestration and reports.
 
-## Consuming Metadata in Landing Zones
+## Tag Schemas
+
+[Configured tag-schemas](meshstack.tag-schema.md) describe the tags available on each meshModel entity and
+their associated validation rules. These rules can describe for example if a tag is required or optional.
+
+> meshStack currently supports tag schemas on [meshCustomers](meshcloud.customer.md) and [meshProjects](meshcloud.project.md) only.
+
+For each supported entity, meshStack can maintain two separate separate tag-schemas: **unrestricted** and **restricted** tags.
+
+### Unrestricted Tags
+
+Entity owners can edit tag values for **unrestricted tags** in self-service via meshPanel. For example, a user with a [Customer Admin](./meshcloud.customer.md#assign-meshcustomer-roles) role can edit tags values on the [meshCustomer](meshcloud.customer.md) as well as the [meshProjects](meshcloud.project.md) owned by that customer.
+
+### Restricted Tags
+
+Only [Partners](./administration.index.md) can edit tag values for **restricted tags**. This is useful if an organization
```

Comment:
Good point. But as far as I know partners can not change tag values through the meshPanel in general. Mostly the values can be changes by authoritative external systems.

Reply:
I just copy pasted the stuff :D I am not sure but I change the text accordingly.